Library/Homebrew/diagnostic.rb

```diff
@@ -1435,5 +1420,34 @@ def all
         methods.map(&:to_s).grep(/^check_/)
       end
     end # end class Checks
+
+  end
+
+  def inject_dump_stats!(the_module, pattern = /.*/)
```

Comment:
Yeah, that makes sense: you'd almost never actually want to do `/.*/` because there are so many small methods in most modules and classes, and if you do, it should be explicit. Removed the default and moved it to Utils.